timeseries/src/autogluon/timeseries/models/autogluon_tabular/mlforecast.py

```diff
@@ -80,11 +85,11 @@ def __init__(
 
         self._sum_of_differences: int = 0  # number of time steps removed from each series by differencing
         self._max_ts_length: Optional[int] = None
-        self._target_lags: Optional[List[int]] = None
-        self._date_features: Optional[List[str]] = None
-        self._mlf: Optional[MLForecast] = None
+        self._target_lags: np.ndarray
+        self._date_features: Optional[List[Callable]] = None
```

Comment:
Similar to above, this could this be `self._date_features: List[Callable]` (without `Optional` and `= None`)?